luna/featurevis/featurevis.py

```diff
@@ -171,25 +176,30 @@ def compute_activation(input_image, model, filter_index):
         filter_activation = activation[:, filter_index, :, :]
     else:
         filter_activation = activation[:, :, :, filter_index]
-    return tf.reduce_mean(filter_activation)
+    activation_score = tf.reduce_mean(filter_activation)
+    if regularizers:
+        obj = regs.perform_regularization(activation, regularizers)
+        return obj
```

Comment:
Why is it suddenly called `obj`?

Reply:
you could do:

```
if regularizers:
    activation_score = regs.perform_regularization(activation, regularizers)
return activation_score
```
No need to return twice.

Also, do we only need the `reduce_mean` if we do not use the regularizers?

Reply:
Yes. With the reduce_mean we are calculating the activation value.